### DISCUSSION TITLE
Is it possible to change the data type (DPT) in the configuration? Or select it?

### DISCUSSION THREAD

**Discussion by bbcare** [ascillato/Tasmota_KNX#593] [Ideas]
Is it possible to change the data type (DPT) in the configuration? Or select it?

**Comment by bartdesign** [comment]
+1 for this feature. DPT14 for temperatures is not ideal :-)

**Comment by bbcare** [comment]
With the KNX standard, the data point for
9.001 DPT_Value_Temp or
16.000 DPT_String_ASCII or
16.001 DPT_String_8859_1
If the DPT in tasmota_knx could be set flexibly, this would bring a lot of advantages.

**Comment by bbcare** [comment]
Here, for example, would be the current KNX data point list

[KNX Data Points](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.knx.org/wAssets/docs/downloads/Certification/Interworking-Datapoint-types/03_07_02-Datapoint-Types-v02.02.01-AS.pdf&ved=2ahUKEwiBgKzM3u37AhUzcvEDHYGBBNEQFnoECBEQAQ&usg=AOvVaw1V_kL8xwe95wedZWlBN4bN)
